tracks/nit/nitcc/config.ini

```diff
@@ -1,2 +1,3 @@
 title=Elvish Vectorial Images and nitcc
 req=visitor
+languages=plain
```

Comment:
`plain` is mainly here to test that a mission ini can override the track ini.

However, in this specific mission (from CSG16), the answer was a plain flag (not code).
So, maybe for some real mission we want a direct answer (not code). I used `plain` to explain the UI that we do not want code highlighting.

For the default, I'm not sure what we want: plain? all languages?
Maybe just refuse the load tracks that does not set the languages?